server/conf.c

```diff
@@ -219,6 +225,17 @@ static int parse_upsd_conf_args(size_t numargs, char **arg)
 #endif /* WITH_CLIENT_CERTIFICATE_VALIDATION */
```

Comment:
@aquette : looking at this older code above, should client certs validation only apply to NSS but not OpenSSL?

Reply:
Posted as issue #1096 to follow up later